app/models/user.rb

```diff
@@ -65,24 +65,20 @@ def self.tenant_id_clause(user_or_group)
     {table_name => {:id => users_ids}}
   end
 
-  def self.in_region
+  def self.in_my_region_column
     where(:region => my_region_number)
   end
 
-  def self.in_my_region
```

Comment:
`in_region` already exists in the ar_region.rb, so there's no need to have it here as well...the only reason it's here is to use the region column, which I believe should be removed entirely.  See https://github.com/Fryguy/manageiq/blob/8210a892bd6131b4931fbc409df24ad03a3ec941/lib/extensions/ar_region.rb#L72-L74